src/ophys_etl/modules/segmentation/detect/feature_vector_utils.py

```diff
@@ -0,0 +1,246 @@
+from typing import Tuple, Optional, List
+import numpy as np
+
+
+def choose_extreme_pixels(
+        image_data: np.ndarray,
+        delta_z: List[float],
```

Comment:
since the mean value doesn't seem to come into play here, reference z-score in this argument name is a little confusing. Shouldn't this be `delta_sigma`?

Reply:
fixed